src/routes/namespaces/[namespace]/workflows/index.svelte

```diff
@@ -14,53 +17,45 @@
     const executionStatus = page.query.get('status') as WorkflowStatus;
     const query = page.query.get('query');
 
-    const parameters: FilterParameters = {
+    const parameters: ValidWorkflowParameters = {
       workflowId,
       workflowType,
       timeRange,
       executionStatus,
       query,
     };
-    const initialData = await fetchAllWorkflows(namespace, parameters, fetch);
+
+    const workflows = await fetchAllWorkflows(namespace, parameters, fetch);
 
     return {
-      props: { initialData, namespace, parameters, isAdvancedSearch },
+      props: { workflows, namespace, parameters, isAdvancedSearch },
     };
   }
 </script>
 
 <script lang="ts">
   import VirtualList from '@sveltejs/svelte-virtual-list';
 
-  import { fetchAllWorkflows } from '$lib/services/workflow-service';
-  import { refreshable } from '$lib/stores/refreshable';
-
   import WorkflowsSummaryTable from './_workflows-summary-table.svelte';
   import WorkflowsSummaryRow from './_workflows-summary-row.svelte';
   import WorkflowFilters from './_workflow-filters.svelte';
   import EmptyState from '$lib/components/empty-state.svelte';
   import WorkflowsLoadingState from './_workflows-loading.svelte';
 
   export let namespace: string;
-  export let initialData: ReturnType<typeof fetchAllWorkflows>;
-  export let parameters: FilterParameters;
+  export let workflows: CombinedWorkflowExecutionsResponse;
   export let isAdvancedSearch: boolean;
 
   let timeFormat: TimeFormat = 'UTC';
 
   const errorMessage = isAdvancedSearch
     ? 'Please check your syntax and try again.'
     : 'If you have filters applied, try adjusting them.';
-
-  $: data = refreshable(
-    () => fetchAllWorkflows(namespace, parameters),
-    initialData,
-  );
 </script>
 
 <h2 class="text-2xl">Workflows</h2>
 <WorkflowFilters bind:timeFormat />
-{#await $data}
+{#await workflows}
```

Comment:
Dropping refreshable makes a lot of this feel better, glad to see it work out so well